ui/src/app/pages/homepage/data-access-requirements.spec.tsx

```diff
@@ -335,6 +336,9 @@ describe('DataAccessRequirements', () => {
         });
 
         expect(findCompletionBanner(wrapper).exists()).toBeTruthy();
+
+        // reset to standard behavior for other tests
+        jest.useRealTimers();
```

Comment:
Actually it's probably safer to move this to an `afterEach`. If this test fails midway, it won't run this line